content/teaching/python-mooc/week1_assignment_questions.md

```diff
@@ -1,6 +1,9 @@
-# Python for Beginners
-
-## Week 1 Assignment (Part 1 - Questions)
+---
+title: "Week 1 Assignment (Part 1 - Questions)"
+language: "en"
+published: true
+tags: ["FH Aachen", "Thesis"]
+---
 
 ### Question 1
```

Comment:
Header level should be decreased to 2 (i.e. ## Question 1). Also true for following headlines.